### PR TITLE
Fixing errors 204 "Lines should be no longer than 160 chars"

### PR DESCRIPTION
**Pull request by nhosoi**
Deleting extra molecule files INSTALL.rst, playbook.yml & yaml-lint.yml

This is part of collection conversion preparation to avoid having
an extra option in molecule/default/molecule.yml.

### CODE REVIEW

File: roles/rsyslog/tasks/outputs/elasticsearch/main.yml [hunk 1]
```diff
@@ -27,7 +27,10 @@
 
     - name: Check certs - use_cert is true, but triplets are not given
       fail:
-        msg: "Error: you specified use_cert: true; you must specify all 3 of ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src, private_key_src, or set use_cert: false in the elasticsearch output named {{ item.name }}"
+        msg: "Error: you specified use_cert: true; you must specify all 3 of \
```
Comment (richm) [inline roles/rsyslog/tasks/outputs/elasticsearch/main.yml:30]: I suggest using the yaml multiline string operator `>-` here:
```yaml
msg: >-
  Error: you specified use_cert: true; you must specify all 3 of
  ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src,
  private_key_src, or set use_cert: false in the elasticsearch output
  named {{ item.name }}
```
the `>` will concatenate all of the lines into a single line for output, with each newline in the input being replaced by a space in the output - the `-` means do not add a trailing newline or space to the output

File: roles/rsyslog/tasks/outputs/elasticsearch/main.yml [hunk 2]
```diff
@@ -50,7 +53,8 @@
 
     - name: Check certs - key/certs data are provided, but use_cert is false
       fail:
-        msg: "Error: you specified use_cert: false and also specified one or more cert/key files in the elasticsearch output named {{ item.name }}"
+        msg: "Error: you specified use_cert: false and also specified one or \
+          more cert/key files in the elasticsearch output named {{ item.name }}"
```
Comment (richm) [inline roles/rsyslog/tasks/outputs/elasticsearch/main.yml:57]: see above

File: tests/tests_basics_forwards_cert_missing.yml [hunk 3]
```diff
@@ -19,7 +19,9 @@
   vars:
     __test_key_name: test-key.pem
     __test_key: /tmp/{{ __test_key_name }}
-    __expected_error: "Error: tls is enabled in forwards_severity_and_facility; you must specify logging_pki_files ca_cert_src and/or ca_cert in the playbook var section."
+    __expected_error: "Error: tls is enabled in forwards_severity_and_facility; \
```
Comment (richm) [inline tests/tests_basics_forwards_cert_missing.yml:22]: see above

File: tests/tests_files_elasticsearch_certs_incomplete.yml [hunk 4]
```diff
@@ -36,7 +36,9 @@
     __test_ca_cert: /tmp/es-ca.crt
     __test_cert: /tmp/es-cert.pem
     __test_key: /tmp/es-key.pem
-    __expected_error: "Error: you specified use_cert: true; you must specify all 3 of ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src, private_key_src, or set use_cert: false in the elasticsearch output named elasticsearch_output"
+    __expected_error: "Error: you specified use_cert: true; you must specify all 3 of \
```
Comment (richm) [inline tests/tests_files_elasticsearch_certs_incomplete.yml:39]: see above

File: tests/tests_files_elasticsearch_use_cert_false_with_keys.yml [hunk 5]
```diff
@@ -30,7 +30,8 @@
     __test_ca_cert: /tmp/es-ca.crt
     __test_cert: /tmp/es-cert.pem
     __test_key: /tmp/es-key.pem
-    __expected_error: "Error: you specified use_cert: false and also specified one or more cert/key files in the elasticsearch output named elasticsearch_output"
+    __expected_error: "Error: you specified use_cert: false and also specified one or \
```
Comment (richm) [inline tests/tests_files_elasticsearch_use_cert_false_with_keys.yml:33]: see above

File: tests/tests_files_elasticsearch_use_local_cert_nokeys.yml [hunk 6]
```diff
@@ -28,7 +28,9 @@
   become: true
   vars:
     __test_inputfiles_dir: /var/log/inputdirectory
-    __expected_error: "Error: you specified use_cert: true; you must specify all 3 of ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src, private_key_src, or set use_cert: false in the elasticsearch output named elasticsearch_output"
+    __expected_error: "Error: you specified use_cert: true; you must specify all 3 of \
```
Comment (richm) [inline tests/tests_files_elasticsearch_use_local_cert_nokeys.yml:31]: see above

**Comment by nhosoi** [comment]
Thank you, @richm.
It seems the logging role has more places violating the line length 80... (;_;)
https://travis-ci.org/github/linux-system-roles/logging/jobs/717466366

**Comment by richm** [comment]
> Thank you, @richm.
> It seems the logging role has more places violating the line length 80... (;_;)
> https://travis-ci.org/github/linux-system-roles/logging/jobs/717466366

oh my - that is a lot - perhaps better to not worry about the line length?  I think it is a low priority

also, you can run `tox -e yamllint` locally to see the errors, but I wouldn't worry about it

**Comment by nhosoi** [comment]
> > Thank you, @richm.
> > It seems the logging role has more places violating the line length 80... (;_;)
> > https://travis-ci.org/github/linux-system-roles/logging/jobs/717466366
> 
> oh my - that is a lot - perhaps better to not worry about the line length? I think it is a low priority
> 
> also, you can run `tox -e yamllint` locally to see the errors, but I wouldn't worry about it

Yeah, `tox -e molecule` also issues the errors... I was hoping to make the logging role error free as much as possible to prepare to run some tests in the collections environment. I've finished some files already. If ok, I'd like to continue...

**Comment by richm** [comment]
> > > Thank you, @richm.
> > > It seems the logging role has more places violating the line length 80... (;_;)
> > > https://travis-ci.org/github/linux-system-roles/logging/jobs/717466366
> > 
> > 
> > oh my - that is a lot - perhaps better to not worry about the line length? I think it is a low priority
> > also, you can run `tox -e yamllint` locally to see the errors, but I wouldn't worry about it
> 
> Yeah, `tox -e molecule` also issues the errors... I was hoping to make the logging role error free as much as possible to prepare to run some tests in the collections environment. I've finished some files already. If ok, I'd like to continue...

OK - but concerning the collections work - I think we're going to have to figure out how to support some differences in yamllint, ansible-lint, flake8, etc. behavior among the roles.  For example, if we have to disable yamllint line length checking for all of the roles, I think that's ok for now, if it helps us to migrate to collections faster.

**Comment by nhosoi** [comment]
Low priority. Closing for now.

**Pull request closed without merging**